### DISCUSSION TITLE
Question about extension

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [darkreader/darkreader#12629] [Q&A]
Is there a setting so I can only make specific sites dark/light mode?

**Comment by deus-e-x** [comment]
When on a website where you want to disable Darkreader from applying its theming, press the keyboard combo "Alt+Shift+A" to disable Darkreader permanently for that specific webpage.

To reenable Darkreader for that webpage, press "Alt+Shift+A" again to toggle Darkreader back on.

**Comment by Fireworkstars46** [comment]
So I have to use a keyboard shortcut and not manually clicking it off for it?

**Reply by deus-e-x** [reply]
You can do it either way you prefer.

Using the mouse you have to navigate your mouse cursor to the Dark Reader icon on the top, click the Dark Reader icon once to access the options, then navigate the cursor over the current webpage field and click it to toggle the Dark Reader service on/off for that website.

The keyboard shortcut allows you to toggle Dark Reader on/off with the same key combo, which you can adjust to your preferred key combination, the default is "Alt-Shift+A".
